### ISSUE TITLE
How to apply pcv.naive_bayes_classifier output as mask

### ISSUE THREAD

**Issue by km4htc** [danforthcenter/plantcv#658]
Not exactly a bug, per se, just a best practice question: once I've got the output from pcv.naive_bayes_classifier, is there a simple way to apply the dictionaries as a mask to the original image? Right now the only workaround I've found is to use the colorize function, save a binary image to file, read the image back in and then apply it as a mask... seems like I'm missing a better way to access the info in the dictionaries.

```
# Read image
img, path, filename = pcv.readimage("original-img.jpg")

# Classify each pixel as plant or background 
classified = pcv.naive_bayes_classifier(rgb_img=img, pdf_file="classifier.txt")

# Create a binary image, print to "test.png", then read test.png back in to pcv
mask = pcv.visualize.colorize_masks(masks=[classified['plant'], classified['background']], 
                                               colors=['white', 'black'])

pcv.print_image(mask, "test.png")

mask, path, filename = pcv.readimage("test.png")

# Apply "test.png" as ask to original img
masked = pcv.apply_mask(img=img, mask=mask, mask_color='white')
```

In short I'd like to avoid the intermediate steps of writing the mask I want out to file and then back in again.

Thanks in advance for any solutions!

**Comment by km4htc** [comment]
Sorry all... I'd been staring at this for too long apparently. I was able to accomplish what I wanted by the following:

`# Read image and rotate
img, path, filename = pcv.readimage("original-img.JPG")

# Classify each pixel as plant or background (background and system components)
classified = pcv.naive_bayes_classifier(rgb_img=img, pdf_file="classifier.txt")

# Create a binary image and print to file
mask = pcv.visualize.colorize_masks(masks=[classified['plant'], classified['background']], 
                                               colors=['white', 'black'])

masked = pcv.apply_mask(img=img, mask=mask, mask_color='white')`

**Comment by nfahlgren** [comment]
Nice, no problem @km4htc!

If the plant mask alone suffices you could skip the intermediate `pcv.visualize.colorize_masks` step (if you want) and directly do:

```python
img, path, filename = pcv.readimage("original-img.JPG")

# Classify each pixel as plant or background (background and system components)
classified = pcv.naive_bayes_classifier(rgb_img=img, pdf_file="classifier.txt")

# Mask the image with the plant mask
masked = pcv.apply_mask(img=img, mask=classified["plant"], mask_color='white')
```

**Comment by km4htc** [comment]
Ah that makes sense how you can call just the "plant" or "background" portions of the dict. I've been taking a sort of learning by doing approach to python and have really appreciated all the work you the plantcv team have done to document everything so well! Thanks @nfahlgren !!
